locales/messages.json

```diff
@@ -1115,10 +1115,12 @@
   "_UnsupportedPort.comment": "An example of {package_name} is zlib.",
   "UnsupportedPortDependency": "- dependency {value} is not supported.",
   "_UnsupportedPortDependency.comment": "'{value}' is the name of a port dependency.",
-  "UnsupportedPortFeature": "{spec} is only supported on '{supports_expression}'",
-  "_UnsupportedPortFeature.comment": "An example of {spec} is zlib:x64-windows. An example of {supports_expression} is windows & !static.",
   "UnsupportedShortOptions": "short options are not supported: '{value}'",
   "_UnsupportedShortOptions.comment": "'{value}' is the short option given",
+  "UnsupportedSupportsExpression": "{spec} is only supported on '{supports_expression}'. This usually means that there are known build failures, or runtime problems, when building other platforms. To ignore this and attempt to build {spec} anyway, rerun vcpkg with `--allow-unsupported`.",
+  "_UnsupportedSupportsExpression.comment": "An example of {spec} is zlib:x64-windows. An example of {supports_expression} is windows & !static.",
```

Comment:
```suggestion
  "_UnsupportedSupportsExpression.comment": "An example of {spec} is zlib:x64-windows. An example of {supports_expression} is 'windows & !static'.",
```

Reply:
This (and all other examples) are generated code.